### PR TITLE
Project clean up

### PR DESCRIPTION
**Pull request by DexterWurnWSU**
This cleans up our Read me and makes a file so that it is easier to connect Pull Requests to Actions/Tasks

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -113,6 +89,21 @@ the client.
 - Backquote (`) to write text on any block (signs).
 - Arrow keys emulate mouse movement.
 - Enter emulates mouse click.
+- ESC closes game down.
+- F2 opens in game clock.
```
Comment (Bravo-0-29er) [inline README.md]: F3 instead of F2

File: README.md [hunk 2]
```diff
@@ -152,65 +143,13 @@ Teleport to the specified chunk.
...
-
-Collision testing simply adjusts the player’s position to remain a certain distance away from any adjacent blocks that are obstacles. (Clouds and plants are not marked as obstacles, so you pass right through them.)
-
-#### Sky Dome
-
-A textured sky dome is used for the sky. The X-coordinate of the texture represents time of day. The Y-values map from the bottom of the sky sphere to the top of the sky sphere. The player is always in the center of the sphere. The fragment shaders for the blocks also sample the sky texture to determine the appropriate fog color to blend with based on the block’s position relative to the backing sky.
-
-#### Ambient Occlusion
-
-Ambient occlusion is implemented as described on this page:
```
Comment (Bravo-0-29er) [inline README.md:211]: All of this that was deleted from the original fork probably could have been left, however if we only want the readme to reflect what our contributions were, I understand deleting this information.

**Review by Bravo-0-29er — COMMENTED** [review]
Overall the revision of the ReadMe looks much cleaner and more relevant to what our contributions to this project were, however I do not know if we want to remove as much of the original ReadMe as what is reflected now.

**Review by Bravo-0-29er — APPROVED** [review]
This new file is a very good addition to better show what contributions were made by which member.

File: README.md (hunk 1 above)
Comment (DexterWurnWSU) [reply README.md]: will fix this

Comment (DexterWurnWSU) [reply README.md]: fixed

**Comment by BrodyEngle-Burdette** [comment]
Good changes, Its always a good idea to keep track of what all our key binds do. All the info is relevant in the ReadMe as well, but the tasks file is wrong, I made  the tasks 1.1, 1.2, and 1.3 from the user stories

**Comment by rileymoore4521** [comment]
I like the task document a lot. It makes it easier to know whos done what and what is actually being done. This can help for later commits from other members.

**Pull request merged**